.github/workflows/commit-check.yml

```diff
@@ -0,0 +1,12 @@
+name: Commit Checks
+on:
+  pull_request:
+
+jobs:
+  no-merge-commits-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
We generally use v4 everywhere. It's worth checking whether that can work instead.

Reply:
Ah, yep. Copy/paste job. Should be fixed now. 😅